pkg/gitlab/version.go

```diff
@@ -0,0 +1,22 @@
+package gitlab
+
+type GitLabVersion struct {
+	Major  int
+	Minor  int
+	Patch  int
+	Suffix string
+}
+
+// PipelineJobsKeysetPaginationSupported returns true if the GitLab instance
+// is running 15.9 or later.
+func (v GitLabVersion) PipelineJobsKeysetPaginationSupported() bool {
```

Comment:
should we make this per call specific if additional ones get implemented in future versions? 🤔

Reply:
I think so because keyset pagination for different endpoints vary depending on the version. For example, `ListUserProjects` in https://docs.gitlab.com/ee/api/projects.html#list-user-projects needs 14.10. Were you thinking we should use a single function name with some feature parameter?

Reply:
yes, let's merge it as is though, I think we can add this capability separately 👍